### ISSUE TITLE
Chrome: Keyboard focus outline missing sides on several elements in email picker

### ISSUE THREAD

**Issue by shane-tomlinson** [mozilla/persona#1583]
https://skitch.com/set117/83xex/screen-shot-2012-05-11-at-11.02.15-am

**Comment by shane-tomlinson** [comment]
Fixed with redesign.  Closing.
